forge/postoffice/index.js

```diff
@@ -106,15 +133,15 @@ module.exports = fp(async function (app, _opts, next) {
                     }
                 })
             }
-            if (app.config.email.debug) {
-                app.log.info(`
-    -----------------------------------
-    to: ${mail.to}
-    subject: ${mail.subject}
-    ------
-    ${mail.text}
-    -----------------------------------`)
-            }
+        }
+        if (isConfigured && app.config.email.debug) {
```

Comment:
By adding `isConfigured` here, it is no longer possible to enable debug mode without also providing a valid smtp/transport/ses configuration. That is not the intended behaviour - if you're developing locally it should be possible to enable email in debug mode without requiring an external email config to match.

Reply:
I can address that, however, for info, I did this since prior to this modification if email transport failed, EMAIL_ENABLED would get toggled off and no debug was possible at all (settings were essentially mutually exclusively e.g. have "EMAIL_ENABLED and zero config" or "working config" )

I can certainly re-address this & make debug independent of configuration or SMTP status if thats preferrable?.

Reply:
Addressed in [e4334c1](https://github.com/flowforge/flowforge/pull/1299/commits/e4334c10484f5936659886f0bc393e4bd6ae89b5)